route/option.go

```diff
@@ -328,6 +328,28 @@ func Sorter(
 	return sorterFunc(sorter)
 }
 
+// PlanMarshaller defines an interface that is used to override router's default
+// output when using multiple vehicles.
+type PlanMarshaller interface {
```

Comment:
`Marshaller` -> `Marshaler`

Reply:
was removed in favor of other implementation